codespell_lib/data/dictionary.txt

```diff
@@ -6048,15 +6111,19 @@ deparmental->departmental
 deparments->departments
 depdencies->dependencies
 depdency->dependency
-depdendence->dependence
```

Comment:
Why deleting this?

Reply:
Thanks, not sure how. Fixed in ae20bad (also added 2 more entries)